lldb/scripts/generate-sbapi-dwarf-enum.py

```diff
@@ -37,6 +38,10 @@ def emit_enum(input, output):
     with open(input, "r") as f:
         lines = f.readlines()
 
+    # Create output folder if it does not exist
+    if not os.path.isdir(os.path.dirname(output)):
```

Comment:
This can be simplified: https://docs.python.org/3/library/os.html
```suggestion
        os.makedirs(os.path.dirname(output), exist_ok=True)
```